src/main/java/org/spongepowered/common/data/SpongeManipulatorRegistry.java

```diff
@@ -327,7 +327,11 @@ private SpongeManipulatorRegistry() {
     public ValueProcessor<?, ?> getDelegate(Key<?> key) {
         if (this.tempRegistry != null) {
             // During soft registrations
-            return new ValueProcessorDelegate(key, ImmutableList.copyOf(this.tempRegistry.valueProcessorMap.get(key)));
+            if (this.tempRegistry.valueProcessorMap.containsKey(key)) {
```

Comment:
Make it faster simply by calling `get` once and checking if the processor is null.

Reply:
fixed